src/network/packet/AvailableCommandsPacket.ts

```diff
@@ -45,7 +45,7 @@ export default class AvailableCommandsPacket extends DataPacket {
 
             // Flags
             this.writeByte(data.flags);
-            this.writeByte(data.permission);
+            this.writeByte(data.permission as any);
```

Comment:
Should be numeric?

Reply:
This might be a problem on jsbinaryutils end.